### DISCUSSION TITLE
Is it means for really serious purpose? or shall we not using mother to description the Silk.NET project?

### DISCUSSION THREAD

**Discussion by ghost** [dotnet/Silk.NET#585] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/278153/129870200-30866135-a2b5-464d-a0b0-24c09e8ef5f0.png)
Is it a joke ? or what is the real meaning about the words?

**Comment by Perksey** [comment, marked as answer]
Yes it is a joke. Please read the readme to find out more about the project.

**Reply by ghost** [reply]
Thanks for explain, I read it .
